src/Common/src/System/Security/Cryptography/Asn1/GeneralNameAsn.cs

```diff
@@ -47,7 +46,7 @@ internal struct GeneralName
     }
 
     [StructLayout(LayoutKind.Sequential)]
-    internal struct OtherName
+    internal struct OtherNameAsn
     {
         internal string TypeId;
```

Comment:
This is missing `[ObjectIdentifier]`. I will add it along with the ASN definitions in comment.